sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mlme.c

```diff
@@ -273,16 +259,11 @@ int _fopts_mlme_link_check_req(lorawan_buffer_t *buf)
     return GNRC_LORAWAN_CID_SIZE;
 }
 
-static int _mlme_link_check_ans(gnrc_lorawan_t *mac, lorawan_buffer_t *fopt)
+static int _mlme_link_check_ans(gnrc_lorawan_t *mac, uint8_t *p)
```

Comment:
can be void, as you have only a single return value, i.e. `0`, and also not use it below when calling/exec the function

Reply:
done! (https://github.com/RIOT-OS/RIOT/pull/11022/commits/22aa1aee7f670a164f742704f801cd1df6e6ab4c)